spec/system/notifications/notifications_page_spec.rb

```diff
@@ -85,4 +85,58 @@ def interact_with_each_emojis
       validate_reply(comment.id)
     end
   end
+
+  context "with welcome notifications" do
+    let(:mascot_account) { create(:user) }
+
+    before do
+      allow(Notification).to receive(:send_welcome_notification).and_call_original
+      allow(User).to receive(:mascot_account).and_return(mascot_account)
+      SiteConfig.staff_user_id = mascot_account.id
```

Comment:
You could stub this value so you dont have to worry about state before and after.

Reply:
Nice idea, done! ✅